packages/apps-script-portfolio-tracker-sample/src/sheet-utils.ts

```
@@ -0,0 +1,383 @@
+  getRange(): GoogleAppsScript.Spreadsheet.Range | undefined {
+    const gridRange = this.state.gtable.range;
+    return this.getRangeForColumns(
+      0,
+      gridRange.endColumnIndex - gridRange.startColumnIndex,
+    );
+  }
+
+  // startColumnIndex is 0-based relative to the table.
+  private getRangeForColumns(startColumnIndex: number, numColumns: number) {
+    const gridRange = this.state.gtable.range;
+    if (this.getNumRows() == 0) {
```

Biome (2.5.0)

[error] Using == may be unsafe if you are relying on type coercion.

(lint/suspicious/noDoubleEquals)